README.md

```diff
@@ -1,18 +1,12 @@
-# [`agilex-limo` module](https://app.viam.com/module/viam/agilex-limo)
-
-This module implements the [`rdk:component:base` API](https://docs.viam.com/components/base/#api) in an `agilex` model for the [AgileX LIMO](https://global.agilex.ai/education/4) base to be used with [`viam-server`](https://docs.viam.com/). This driver supports differential, ackermann, and omni directional steering modes over the serial port.
```

Comment:
This section [remains](https://github.com/viamrobotics/docs/blob/b2ded2e8193053a4a714500d6ac652ee084e0a94/docs/registry/create/_index.md?plain=1#L1167-L1169) in the template, should it remain here?

Reply:
Oh yeah sorry oops!